src/transporters.ts

```diff
@@ -69,17 +72,17 @@ export default class DefaultTransporter {
       // responds without proper content-type.
       try {
         body = JSON.parse(body);
-      } catch (err) { /* no op */ }
+      } catch (err) { /* no op */
```

Comment:
I have added a manual fixup for this. Please open bugs against clang-format upstream the style-guide.